html2docx.py

```diff
@@ -10,6 +10,16 @@
 WHITESPACE_RE = re.compile(r"\s+")
 
 
+def get_attrs(attrs: List[Tuple[str, Optional[str]]]) -> List[str]:
+    args = []
+    for attr in attrs:
+        if attr == ("style", "text-decoration: underline;"):
+            args.append("underline")
+        elif attr == ("style", "text-decoration: line-through;"):
```

Comment:
The style attribute may not always have exactly this whitespace or formatting. It could also contain multiple inline styles. I think we'll need to some basic CSS parsing. Perhaps an existing library already does this. More tests for these edge cases would be nice.